core/templates/dev/head/domain/classroom/ClassroomBackendApiService.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to get topic data.
```

Comment:
topic -> classroom

Reply:
Let's fix this in the next PR -- I have added a comment in #7507. Thanks @vojtechjelinek!

Reply:
Done